### ISSUE TITLE
trim-alt-alleles error

### ISSUE THREAD

**Issue by dwaggott** [samtools/bcftools#256]
I got this when trying to apply the tim alleles argument to a vcf of multiallelic variants, many of which had AC=0. I can upload a sample vcf if this isn't reproducible.

```
 bcftools view -a multiallelic.vcf.gz > multiallelic_trim.vcf   
bcftools: vcfutils.c:578: bcf_remove_alleles: Assertion `nori==nA_ori' failed.
Aborted
```

**Comment by mcshane** [comment]
Can you upload you sample VCF?

**Comment by niruchennagiri-claritas** [comment]
Did this get resolved? I see the same bug now. The culprit is line:
chr1	5934837	.	C	T	1117.63	PASS	AF=0.976378;AO=121;DP=131;FAO=124;FDP=127;FR=.;FRO=3;FSAF=50;FSAR=74;FSRF=2;FSRR=1;FWDB=0.0241805;FXX=0.030532;HRUN=4;LEN=1;MLLD=128.841;OALT=T;OID=.;OMAPALT=T;OPOS=5934837;OREF=C;PB=0.5;PBP=1;QD=35.201;RBI=0.0514354;REFB=0.00491583;REVB=0.0453972;RO=2;SAF=50;SAR=71;SRF=1;SRR=1;SSEN=0;SSEP=0;SSSB=-0.00276194;STB=0.506448;STBP=0.411;TYPE=snp;VARB=0.00123751	GT:AF:AO:DP:FAO:FDP:FRO:FSAF:FSAR:FSRF:FSRR:GQ:RO:SAF:SAR:SRF:SRR	1/1:0.976378:121:131:124:127:3:50:74:2:1:33:2:50:71:1:1

I can upload a sample vcf if needed.

**Comment by mcshane** [comment]
No, this was not resolved. Can you send your sample VCF including the header?

**Comment by niruchennagiri-claritas** [comment]
I would like to avoid sharing the VCF file publicly. Can you provide a
direct email address that I can send the VCF file to?
Thanks.

**Comment by mcshane** [comment]
My email address is on my profile page. Alternatively, just post the header that goes with the above record here.

**Comment by tommycarstensen** [comment]
@niruchennagiri-claritas I solved the problem by piping to `bcftools annotate -x FORMAT,INFO` prior to piping to `bcftools view --trim-alt-alleles` as suggested in [issue 404](https://github.com/samtools/bcftools/issues/404).

**Comment by mcshane** [comment]
This should be fixed via samtools/htslib@ee4725892ee3f29a813ba906d7c8b07928a3f352
